### PR TITLE
Allow camera and microphone for embeddable (v2)

### PR DESCRIPTION
**Pull request by whoaitsaimz**
closes sparkletown/internal-sparkle-issues#662

Trying again with proper base and fresh commits, will close previous PR that had conflicts

### CODE REVIEW

**Comment by whoaitsaimz** [comment]
For reference, previous PR was https://github.com/sparkletown/sparkle/pull/1543

**Comment by whoaitsaimz** [comment]
Oh and -- closes sparkletown/internal-sparkle-issues#662

**Comment by sunny-viktoryia** [comment]
@0xdevalias @mike-lvov do we want these changes to be in the `env/github` only? I guess we could put that to `staging`. (at least the new `const` in the settings).

**Comment by whoaitsaimz** [comment]
@sunny-viktoryia there was a discussion around security, so I was only targeting env/github to start, but happy for this to be available to larger code base. At minimum, if we get it to env/github in the next deploy that would be amazing.  As Mike is still OOO, I think @0xdevalias would again be the one to approve this.

File: src/settings.ts [hunk 1]
```diff
@@ -157,6 +157,8 @@ export const VALID_URL_PROTOCOLS = ["http:", "https:"];
 
 export const IFRAME_ALLOW =
   "accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture; fullscreen";
+export const IFRAME_ALLOW_ADVANCED =
+  "accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture; fullscreen; camera; microphone";
```
Comment (0xdevalias) [inline src/settings.ts:161]:
```suggestion
export const IFRAME_ALLOW_ADVANCED = `${IFRAME_ALLOW} camera; microphone`;
```

**Comment by 0xdevalias** [comment]
@sunny-viktoryia Can be deployed directly to `env/github` for now if needs be. Quicker, and we don't have to think about the security implications to the wider platform. We can bring it back into `staging`/etc later on.

**Comment by 0xdevalias** [comment]
FYI: `env/github` is broken at the moment as per https://doesitsparkle.slack.com/archives/C01ER4M61TL/p1624580015339300

So there's no point deploying this there till that is resolved IMO.

**Comment by malaikaaiyar** [comment]
lgtm!

**Review by malaikaaiyar — APPROVED** [review]

**Comment by whoaitsaimz** [comment]
Does ready for QA (in staging) mean it was not actually merged to the environment, yet...? Id like to understand this process ...when it will be fully deployed?

**Comment by malaikaaiyar** [comment]
Nope, this was merged to env/github because that's the branch that this pr is on. If you look at the top under the title it says that I merged 4 commits into `env/github` from `whoaitsaimz/embeddable`

**Comment by 0xdevalias** [comment]
@whoaitsaimz thats a deficiency in our old sprint board layout/etc because we’re not following our usual flows. Normally a PR getting merged puts it into staging, and needs to be deployed, thus our default automation rules update the column as such. Given we’re bypassing our patterns for these custom environment hacks, those default flows don’t always align to reality.

**Comment by whoaitsaimz** [comment]
There's a missing ; after fullscreen that may be breaking this

Here's the demo venue to check against https://github-summit.com/in/outsnappeddemo

**Review by jonboutelle — COMMENTED** [review]
LGTM

**Comment by 0xdevalias** [comment]
> LGTM

@jonboutelle This was already approved and merged..

The PR @whoaitsaimz was asking about was https://github.com/sparkletown/sparkle/pull/1701, which I approved/merged this morning: https://github.com/sparkletown/sparkle/pull/1701#pullrequestreview-695591899

**Pull request merged**